syllabus/curricula/sequential.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, curriculum_list: List[Curriculum], stopping_conditions: List[
         if len(curriculum_list) == 1:
             warnings.warn("Your sequential curriculum only containes one element. Consider using that element directly instead.")
 
-        self.curriculum_list = self._parse_curriculum_list(curriculum_list)
+        self.curriculum_list = self._parse_curriculum_list(curriculum_list, warmup_strategy=self.warmup_strategy, warmup_samples=self.warmup_tasks)
```

Comment:
I don't think this is the approach we want to take. If people want to add individual startup behaviors to each curriculum in the sequence, they can initialize curricula however they feel and add them to the list. I actually think it would be better to either disallow startup sampling for sequential curricula or use the default implementation (the same way it works in the base Curriculum class).